src/components/Square.js

```diff
@@ -1,24 +1,32 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import './Square.css'
+import './Square.css';
 
 const Square = (props) => {
-  // For Wave 1 enable this 
-  //  Component to alert a parent 
-  //  component when it's clicked on.
+  const onSquareClicked = () => {
+    const updatedSquare = {
+      id: props.id,
+    };
+    props.onClickCallback(updatedSquare);
```

Comment:
If the click handler was defined so that it just expected the id, then wrapping the id in an object wouldn't be necessary.